templates/support/RetryConfiguration.swift.hbs

```diff
@@ -0,0 +1,31 @@
+//  
+//  {{>frag/generatedBy}}
+//
+
+import Foundation
+
+/// A configuration for retrying requests
+public struct RetryConfiguration {
+    /// The maximum number of attempts to make
+    let maxAttempts: Int
+
+    /// The minimum delay between attempts
+    let delay: TimeInterval
+
+    /// The factor by which to scale the delay between attempts (exponential backoff)
+    let scaleFactor: Double
+
+    /**
+    * Initialize a new RetryConfiguration
```

Comment:
I don't know if this is acceptable indentation for a multiline comment in Swift.